libtransmission/port-forwarding-natpmp.cc

```diff
@@ -13,7 +13,7 @@
 #include <fmt/core.h>
 
 #define ENABLE_STRNATPMPERR
-#include <natpmp.h>
+#include "natpmp.h"
```

Comment:
This is the only line I'm not confident with.
Is this really needed?
IMO, 3rd-party libs should be used as system headers, and `SYSTEM_HEADER_SEARCH_PATHS` already includes `libnatpmp`

Reply:
It's the same line as in `port-forwarding-natpmp.h`. And that fixes the build.